source/use-cases/home-resolver.rst

```diff
@@ -1,6 +1,128 @@
+::
+	dig example.com
+
+Setting up for the rest of the network
+--------------------------------------
+
+While we currently have a working instance of unbound, we need it to be reachable from within our entire network. With that comes the headache of dealing with IP addresses. It’s likely that your home router distributed local IP addresses to your devices. If this is the case (i.e. you didn’t change it by hand), the ranges should be between[`RFC1918<http://tools.ietf.org/html/rfc1918>`]:
+::
+	10.0.0.0 - 10.255.255.255 (10/8)
+	172.16.0.0 - 172.31.255.255 (172.16/12)
+	192.168.0.0 - 192.168.255.255 (192.168/16)
+
+The Unbound example config uses the 10.0.0.0/8, so that’s what we use in this example, but note that this can be a source of connectivity errors further on.
+
+Let’s look at a snippet of the example config file. The full example config is almost 1200 lines long, as the capabilities of Unbound are considerable, but we won’t need nearly as much. (If you are interested, any and all configurables can be found in the extensive manual page with *man unbound*)
+
+The example config is found at:
+::
+	/etc/unbound/unbound.conf
+
+if you open this for the first time it looks very empty. It is still usable for one machine, as this is how all the Unbound defaults are configured. It's not, however, enough for our purposes so we will add the minimal configuration options.
+
+The options that we add to the current config file to make it a "minimal usable config" are:
+::
+	server:
+            # the interface that is used to connect to the network, this means on this machine
+            interface: 0.0.0.0
+            interface: ::0
+            # addresses from the IP range that are allowed to connect to the resolver
+            access-control: 10.0.0.0/8 allow
+            access-control: 2001:DB8::/64 allow
+
+The access-control is currently configured to listen to any address on the machine, and only allow queries from the 10.0.0.0/8 IP range.
+
+To prepare our config we are going to modify the existing config in /etc/unbound/unbound.conf. 
+If you open the file we see that there is already an “include” in there. This include enables us to do DNSSEC, which allows Unbound to verify the source of the answers that it receives [LINK ?], so we want to keep this. If you don't have the files that the unclude links to, they can be created using the *unbound-anchor* command.
+
+With your favourite text editor then add the minimal config as shown above, making any changes to the access control where needed. Do note that we strongly recommend to keep the *include* that is already in the file. When you are happy with your config, we first need to kill the currently running unbound server and restart it with our new configuration.
+
+you can kill the current version with 
+::
+	pkill -f unbound
+
+And you can restart Unbound with:
+::
+	unbound -c /etc/unbound.conf
```

Comment:
I would replace those with `unbound-control` commands.
The first is `unbound-control stop`; the second is `unbound-control start`.
That would also be a nice section on how to enable remote control (if not enabled by default on your system).